test/Makefile

```diff
@@ -17,7 +17,7 @@ obj_test_pid_reuse = ${patsubst %.c, %.o, test_pid_reuse.c}
 
 TEST_OBJS = $(sort $(obj_test_deflate) $(obj_test_inflate) $(obj_test_stress) \
     $(obj_test_crc32) $(obj_test_adler32) $(obj_test_multithread_stress))
-EXE = test_inflate test_deflate test_stress test_crc32 test_adler32 \
```

Comment:
If you are removing it here it should be removed from run_tests.sh as well.
Did you had any problem with test_stress? It shouldn't be unsafe unless you drastic reduce the max poll number for wait_for_csb().

Reply:
> Did you had any problem with test_stress?

Yes, when running `make -j$(nproc) check unsafe-check`.
But let me drop this patch for now.  I'll investigate this issue later...